src/main/webapp/css/demo_page-responsive.css

```diff
@@ -68,6 +70,13 @@ and (orientation : landscape) {
         padding: 100px 100px;
         text-align: center;
     }
+    .mobile-box-text {
+        visibility: hidden;
+	height: 0px;
+    }
+    .box image {
```

Comment:
Should this be box-image or box img?

Reply:
oh box-image. makes sense why it wasn't working. i did it another way. let me fix